geth/jail/jail.go

```diff
@@ -22,6 +22,9 @@ const (
 			return new Bignumber(val);
 		}
 	`
+	// ResponseCellCreatedEmpty is returned when cell is successfully created and initialized
+	// but no additional JS was provided to the initialization method.
+	ResponseCellCreatedEmpty = `{"result": ""}`
```

Comment:
Sorry if it was discussed before, but maybe just `EmptyResponse`?
Creating cell is currently the only place where this response is returned, so no need to duplicate "cell created" in error name. Moreover, theoretically this empty response can be used for other actions, so storing "cell created" in it will make even less sense.